pyconvcli/__init__.py

```diff
@@ -0,0 +1 @@
+from .pyconvcli import PyConvCli, ParserArgType
```

Comment:
this exposes PyConvCli and ParserPargType at the root of the projecst so that they can do imports like this
'from pconvcli import PyConvCli,ParserArgType'
instad of
from pyconvcli.poconvcli import  PyConvCli,ParserArgType'